environments/india/proxy.yml

```diff
@@ -5,7 +5,5 @@ J2ME_SITE_HOST: 'j2me-india.commcarehq.org'
 tableau_server: 'tableau2.internal.commcarehq.org'
 TABLEAU_HOST: 'icds.commcarehq.org'
 primary_ssl_env: "india"
-
-# Nginx only accepts on cert, so cat the site cert and chain:
-# $ cat india.commcarehq.org.crt intermediate.crt > india.commcarehq.org.combined.crt
-letsencrypt_cchq_ssl: True
```

Comment:
Why did these get removed in this PR, by the way? I presume this was intentional?

Reply:
Yeah it was intentional. I guess kind of sloppy to do in the same commit, but this is all cruft ever since we changed how proxy certs are handled a while ago.

Reply:
No worries, just wanted to make sure I didn't ignore it in case it wasn't intentional